scripts/resume_arm_time.py

```
@@ -0,0 +1,117 @@
+    body = f"[RESUME DUE] {fire_type} fired for window ending {armed_at}"
+    try:
+        import asyncio
+        from taosmd.service import a2a_send
+        asyncio.run(a2a_send(
+            sender="resume_arm",
+            body=body,
+            thread="agent-rules",
+            data_dir=data_dir,
+        ))
+    except Exception:
+        pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)